packages/server/src/init.ts

```diff
@@ -245,6 +278,8 @@ async function init(config: PadlocConfig) {
         legacyServer
     );
 
+    new DirectorySync(server, directoryProviders);
```

Comment:
I think I'll need to pass the `config.directory` additionally, and add a `scim.url` property to it, to properly build up the URL.